src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/BinderHelper.cs

```diff
@@ -175,12 +175,12 @@ internal static bool IsWindowsRuntimeObject(DynamicMetaObject obj)
                 Type curType = obj.RuntimeType;
                 while (curType != null)
                 {
-                    if (curType.GetTypeInfo().Attributes.HasFlag(System.Reflection.TypeAttributes.WindowsRuntime))
+                    if ((curType.GetTypeInfo().Attributes & System.Reflection.TypeAttributes.WindowsRuntime) != 0)
```

Comment:
I've not checked all of the values to see whether it would matter, but technically I think these should all be of the form ```(source & target) != target``` rather than ```(source & target) != 0```.